torchgeo/transforms/indices.py

```diff
@@ -165,3 +165,39 @@ def __init__(self, index_green: int, index_nir: int) -> None:
             index_nir: index of the Near Infrared (NIR) band in the image
         """
         super().__init__(index_a=index_green, index_b=index_nir)
+
+
+class AppendSWI(AppendNormalizedDifferenceIndex):
+    """Standardized Water-Level Index (SWI).
+
+    If you use this index in your research, please cite the following paper:
+
+    * https://doi.org/10.3390/w13121647
+    """
+
+    def __init__(self, index_red: int, index_swir: int) -> None:
+        """Initialize a new transform instance.
+
+        Args:
+            index_red: index of the VRE1 band, e.g. B5 in Sentinel 2 imagery
+            index_swir: index of the SWIR2 band, e.g. B11 in Sentinel 2 imagery
```

Comment:
We don't add examples for other indices. Should we?

Reply:
Nah I think it is fine